addons/website_event_sale/controllers/main.py

```diff
@@ -59,10 +71,13 @@ def registration_confirm(self, event, **post):
         res = super().registration_confirm(event, **post)
 
         registrations = self._process_attendees_form(event, post)
+        order_sudo = request.website.sale_get_order()
```

Comment:
Answering here as code has changed.

> probably it should be if not any(line.event_ticket_id for line in order_sudo.order_line):

Indeed, my bad 😄 

> it will work but on the other hand quotation will be created for the product existing on the cart

As we don't pass "force_create", I don't think it will affect the sale process in any way.